docs/_includes/footer.html

```diff
@@ -3,10 +3,17 @@
 
     <ul class="icons">
         <li><a href="{{ site.github }}" class="icon circle fa-github"><span class="label">Github</span></a></li>
+        <li><a href="mailto:emodb-open-source@bazaarvoice.com" class="icon circle fa-envelope"><span class="label">Email</span></a></li>
+    </ul>
+
+
+    <ul class="copyright">
+        <li><a href="https://github.com/bazaarvoice/emodb/blob/master/Credits.md">Project Credits</a></li>
```

Comment:
Do we want to keep this page in the github markdown or port it over to the website? I am fine either way.